src/libs/Authentication.ts

```diff
@@ -92,7 +90,7 @@ function reauthenticate(command = '') {
         // Note: It is important to manually set the authToken that is in the store here since any requests that are hooked into
         // reauthenticate .then() will immediate post and use the local authToken. Onyx updates subscribers lately so it is not
         // enough to do the updateSessionAuthTokens() call above.
-        NetworkStore.setAuthToken(response.authToken);
+        NetworkStore.setAuthToken(response.authToken ?? null);
```

Comment:
Most onyx models are loosely typed, meaning that almost anything can be undefined (once the whole codebase is written in Typescript we can improve the types and add better data validation).

`authToken` in most cases (if not for all) indeed a string, I added a fallback just in case - it won't break anything.

Reply:
I am not afraid of the change breaking things, I am afraid that this is making the code more complex and less clear, which is the opposite goal of this migration.
Does this not work if authToken is not set?
In any case, can we make authToken be always string instead of string | null?